pull_into_place/commands/09_compare_best_designs.py

```diff
@@ -504,6 +504,43 @@ def score_value(self, design):
 
 
 
+class ExtraFilterHandler (Metric):
+    """
+    Class that handles filters input by the user. The user should put a "+" or "-"
+    at the beginning of each filter name to define whether larger numbers (+) or
+    smaller numbers (-) are better. Used to make an excel column for
+    extra filters.
+    """
+
+    def __init__(self):
+        align = 'center'
+        num_format = '0.000'
+
+    def get_name(self,filtername):
+
+        if filtername[0] == '+' or filtername[0] == '-':
+            self.color = True
+        else:
+            self.color = False
+
+        self.title = filtername
+        return self.title,self.color
+
+    def load_cell(self, design, verbose=False):
+        setattr(design,self.title,design.structures[self.title][design.rep])
+
+    def face_value(self,design):
+        return getattr(design,self.title)
+
```

Comment:
I don't think you need to implement `load_cell` for this metric because nothing else needs to know about these scores.  Plus `self.title` could have spaces, so it shouldn't really be used for `getattr/setattr` anyway.  I think the best thing would be to just implement `face_value` like this:

    def face_value(self, design):
        return design[self.title][design.rep]